lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -265,17 +265,28 @@ ModuleDependencyVector ClangImporter::bridgeClangModuleDependencies(
       swiftArgs.push_back(IncludeTree);
     }
 
+    // Ensure we get the "real", vfs-remapped path to the modulemap
+    // becuase it is used as an explicit input for the resulting compilation
+    // task.
+    SmallString<128> realModuleMapPath =
```

Comment:
This could be just `StringRef`, right? You could save one potential allocation and just do `realModuleMapPath.str()` below.